definitions/artifacts/azure-storage-account-blob.json

```diff
@@ -18,35 +18,26 @@
       "type": "object",
       "required": [
         "infrastructure",
-        "authentication",
         "security"
       ],
       "properties": {
         "infrastructure": {
           "title": "Infrastructure configuration",
           "type": "object",
           "required": [
-            "ari"
+            "ari",
+            "primary_endpoint",
+            "secondary_endpoint"
```

Comment:
Are both primary/secondary endpoints always created?

Reply:
No, the secondary endpoint becomes the primary endpoint in the event of a failover when geo-replication is enabled. https://learn.microsoft.com/en-us/azure/storage/common/storage-redundancy#read-access-to-data-in-the-secondary-region